core/consensus/babe/src/lib.rs

```diff
@@ -617,9 +622,10 @@ impl<B: Block, C> Verifier<B> for BabeVerifier<C> where
 			.map_err(|e| format!("Could not extract timestamp and slot: {:?}", e))?;
 		let hash = header.hash();
 		let parent_hash = *header.parent_hash();
-		let authorities = authorities(self.client.as_ref(), &BlockId::Hash(parent_hash))
-			.map_err(|e| format!("Could not fetch authorities at {:?}: {:?}", parent_hash, e))?;
-
+		let Epoch { authorities, randomness } =
+			epoch(self.client.as_ref(), &BlockId::Hash(parent_hash))
+			.map_err(|e| format!("Could not fetch epoch at {:?}: {:?}", parent_hash, e))?;
+		let authorities: Vec<_> = authorities.into_iter().map(|(s, _)| s).collect();
```

Comment:
why ignore the weights? easiest thing for the moment is for the runtime to set all weights to `1`, but the consensus-side should still adjust the VRF calculation correctly